### ISSUE TITLE
Hide scout message about event draining

### ISSUE THREAD

**Issue by Oscarchase** [scoutapp/scout_apm_python#498]
When `log_level` is set to `ERROR` and logs are redirected to a log file through `log_file_path`, scout still logs a message like: `Scout draining 1 event for up to 2.0 seconds` on STDOUT, is it possible to remove this message from STDOUT?
Thank you!

**Comment by adamchainz** [comment]
Hi @Oscarchase 

This message is on stderr, and was added recently in version 2.11.0: #473 . We had a bit of debate how to send the message but ultimately copied Sentry, which does the same. If we added a configuration value to disable this, would that suit you?

Btw the `log_level` and `log_file_path` configuration values only affect the core agent, which is a separate process. They don't affect the Python logging configuration for Scout. We have some docs for [configuring Scout's python logging](https://docs.scoutapm.com/#python-logging). #497 will update the names of these configuration values to reduce confusion.

Thanks,

Adam
